apps/vr-tests-react-components/src/stories/Textarea.stories.tsx

```diff
@@ -37,7 +37,11 @@ storiesOf('Textarea Converged', module)
       <Textarea aria-invalid appearance="filled-lighter" placeholder="Placeholder" />
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 9 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27837&commitId=9885293745cdc15969b6279152b21b3238750e38&env=prod)

✅ There was 6 screenshots added, 0 screenshots removed, 1933 screenshots unchanged, 0 screenshots with different dimensions and 3 screenshots with visible difference.


  <details><summary><b>unknown</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Textarea Converged.Disabled.default.chromium.png|114| Changed|
|Textarea Converged.Disabled.focused.chromium.png|114| Changed|
|Textarea Converged.Disabled.hover.chromium.png|114| Changed|
|Textarea Converged.Disabled + filled.default.chromium.png|0| Added|
|Textarea Converged.Disabled + filled.focused.chromium.png|0| Added|
|Textarea Converged.Disabled + filled.hover.chromium.png|0| Added|
|Textarea Converged.Disabled + placeholder.default.chromium.png|0| Added|
|Textarea Converged.Disabled + placeholder.focused.chromium.png|0| Added|
|Textarea Converged.Disabled + placeholder.hover.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>